protocols/sushixswap/contracts/adapters/StargateAdapter.sol

```diff
@@ -133,7 +133,13 @@ abstract contract StargateAdapter is ImmutableState, IStargateReceiver {
         bool failed;
 
         if(gasleft() < reserveGas) {
-            IERC20(_token).safeTransfer(to, amountLD);
+            if(_token != SGETH) {
+                IERC20(_token).safeTransfer(to, amountLD);
+            }
+            /// @dev transfer any native token received as dust to the to address
```

Comment:
Shouldn't this be else?  Also, what if there's no sgeth on a network. We can't set it to address(0) or this will conflcit using address(0) for eth.

Reply:
It doesn't need an else, since if it native token, it will be sent to the user from the transfer below. In case there is no SGETH on that network, it doesn't affect our transaction flow. We don't use SGETH anywhere else apart from here.

Reply:
Gotcha. Token here will never be address(0).